Makefile

```diff
@@ -157,80 +191,63 @@ build-all: $(addprefix bin/dikastes-,$(VALIDARCHES))
 
 .PHONY: build
 ## Build the binary for the current architecture and platform
-build: bin/dikastes-$(ARCH) bin/healthz-$(ARCH)
+build: vendor bin/dikastes-$(ARCH) bin/healthz-$(ARCH)
```

Comment:
why the explicit dependence here? I believe that each binary target already has dependencies on vendor

Reply:
Fixed